src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -617,6 +617,13 @@ private function addValidationSection(ArrayNodeDefinition $rootNode)
                                 ->then(function ($v) { return (string) $v; })
                             ->end()
                         ->end()
+                        ->arrayNode('mappings')
+                            ->prototype('array')
+                                ->children()
+                                    ->scalarNode('file')->end()
```

Comment:
Or if you want to keep them separated, I would use `mappings.files` and `mappings.dirs` or something like that as different lists.